src/components/project_view/submission_detail/ag_test_command_result_detail.vue

```diff
@@ -33,7 +33,7 @@
                         {'actual-return-code-correct': ag_test_command_result !== null
                           && ag_test_command_result.return_code_correct,
                           'actual-return-code-incorrect': ag_test_command_result !== null
-                          && !ag_test_command_result.return_code_correct}]"
+                          && ag_test_command_result.return_code_correct === false}]"
```

Comment:
Can you add a short comment reminding that return_code_correct can be null? Just since this kind of explicit bool check doesn't show up anywhere else in the codebase.

Reply:
@james-perretta  Added in latest commit